src/Agent.Service/Linux/trace.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I don't think we need a trace do we?  See my question about console out.  I believe that's the story for service feedback on both linux and osx